Vagrantfile

```diff
@@ -98,7 +98,7 @@ Vagrant.configure(2) do |config|
 
     config.vm.provider :libvirt do |libvirt|
         config.vm.box = "cilium/ubuntu-16.10"
-        config.vm.box_version = "2.0"
+        config.vm.box_version = "2.1"
```

Comment:
2.1 doesn't exist for libvirt, only `1`

Reply:
I don't have a libvirt setup anymore. Somebody with a libvirt setup will need to build the latest packer scripts.